dependencies.props

```diff
@@ -41,7 +41,7 @@
       <CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
     </RemoteDependencyBuildInfo>
     <RemoteDependencyBuildInfo Include="ProjectKTfs">
-      <BuildInfoPath>$(BaseDotNetBuildInfo)projectk-tfs/$(DependencyBranch)</BuildInfoPath>
+      <BuildInfoPath>$(BaseDotNetBuildInfo)projectk-tfs/release/1.0.0</BuildInfoPath>
```

Comment:
This change looks like a merge issue as I merged it into master yesterday.

Reply:
I suggest rebasing your changes on master.

Reply:
I did rebase and this remained, master has release/1.0.0. Should I change it to the vairable?

Reply:
This matches master but it shouldn't be showing up in your diff if you did a rebase.